docs/easypy/usages.rst

```diff
@@ -295,37 +294,115 @@ constructed and processed using python `argparse`_ module.  Please also see
                                        --clean-devices "[[device_a, device_b, device_c], [device_d, device_e]]"\
                                        --invoke-clean
 
-``--clean-image``
-    specifies images to be used for clean per device
+.. _kleenex_cli_image_format:
+
+``--clean-device-image``
+    specifies images to be used for clean per device.
+
+    .. code-block:: bash
+
+        bash$ pyats run job jobfile.py --testbed-file /path/to/my/testbed.yaml\
+                                       --clean-file /path/to/my/clean.yaml\
+                                       --invoke-clean\
+                                       --clean-device-image PE1:/path/to/clean_image.bin
+
+    This is equivalent to the following in YAML
+
+    .. code-block:: yaml
+
+        images:
```

Comment:
Add the parent keys starting with `devices` so its easier to understand. Same elsewhere